popup/js/popup.js

```diff
@@ -189,38 +215,58 @@ function suggestion() {
     document.getElementById("suggest-label").innerHTML = suggestions[random];
 }
 
+// eslint-disable-next-line vars-on-top
 var theme;
 
 function themeChange() {
-
     theme = localStorage.getItem("theme");
-    if(theme == 'light' )
-        localStorage.setItem( "theme" , "dark" );
-    else
-        localStorage.setItem("theme", "light");
+    // eslint-disable-next-line eqeqeq
+    if (theme == "light") {
+        // eslint-disable-next-line brace-style
+        localStorage.setItem("theme", "dark"); }
+    else { localStorage.setItem("theme", "light"); }
 
+    // eslint-disable-next-line vars-on-top
     var bg = document.getElementById("content");
     bg.classList.toggle("dark");
 
-    var labels = document.getElementsByTagName('label');
+    // eslint-disable-next-line vars-on-top
+    var labels = document.getElementsByTagName("label");
     labels[0].classList.toggle("dark-label");
     labels[1].classList.toggle("dark-label");
     labels[2].classList.toggle("dark-label");
+}
 
+function recordSearchHistory() {
+    var recentSearchQueries = [];
+    if (localStorage.getItem("search")) {
+        // eslint-disable-next-line brace-style
+        recentSearchQueries = JSON.parse(localStorage.getItem("search")); }
+    // eslint-disable-next-line vars-on-top
+    var x = text.value;
+    // eslint-disable-next-line eqeqeq
+    if (x != "" && recentSearchQueries.indexOf(x) == -1) {
+        recentSearchQueries.push(x);
+        localStorage.setItem("search", JSON.stringify(recentSearchQueries));
+    }
 }
 
 
 document.addEventListener("DOMContentLoaded", function () {
-    document.querySelector("button").addEventListener("click", register);
+    document.querySelector("button").addEventListener("click", function () {
+        recordSearchHistory();
+        register();
+    });
     document.addEventListener("keyup", keyboardShortCutListener, false);
     suggestion();
     suggestionAsValue();
-    if (!localStorage.getItem('theme'))
-        localStorage.setItem('theme', 'light');
+    if (!localStorage.getItem("theme")) {
+        // eslint-disable-next-line brace-style
+        localStorage.setItem("theme", "light"); }
 
-    if(localStorage.getItem("theme") == 'dark')
-    {
+    // eslint-disable-next-line eqeqeq
+    if (localStorage.getItem("theme") == "dark") {
```

Comment:
Something like using === instead of == helps us prevent a lot of bugs when applications get big. This one is pretty obvious one. But eslint helps you with bugs that are relatively hard to find. So you should not disable them instead you can look for what does the definition mean and fix it.